src/big/big_float.cr

```diff
@@ -17,6 +18,23 @@ struct BigFloat < Float
     LibGMP.mpf_init_set_str(out @mpf, str, 10)
   end
 
+  def initialize(num : BigInt)
```

Comment:
Either way, I change constructor to do the right thing instead of removing constructor.